content/02.introduction.md

```diff
@@ -13,9 +13,9 @@ Here, we review and discuss one of the most used web-based platforms for computa
 <!-- *Contributors to this section: RCO, SSHS, PHPB* -->
 
 With over 83 million registered users as of 2022, GitHub is the most widely-used web platform for collaborating on computer code [@url:https://github.com/about].
-GitHub provides a simple but powerful web interface that allows users to participate in projects, contribute code, report and discuss code bugs, discover existing code and data, and publish new code.
+GitHub provides a simple but powerful web interface that allows users to participate in projects, contribute code, report, and discuss code bugs, discover existing code and data, and publish new code.
```

Comment:
I think this should be reverted.  "report and discuss code bugs" is one thing.  With the comma it sounds like "report" is an activity.  Alternatively, change to "report bugs, discuss code, "

Reply:
I agree with you! I will change it to "report bugs, discuss code" to make it clearer.

Reply:
Looks like @pedrohbraga made this change so going to resolve comment.